pages.id/common/nvim.md

```diff
@@ -24,7 +24,7 @@
 
 `<Esc>/{{pattern_pencarian}}<Enter>`
 
-- Melakukan penggantian regex pada seluruh berkas:
+- Melakukan penggantian ekspresi reguler pada seluruh berkas:
 
 `<Esc>:%s/{{pattern}}/{{pengganti}}/g<Enter>`
```

Comment:
Please change `pattern` to `regurlar_expresion` here (aswel) (in it's specified translation).